task/ab-load/0.1/ab-load.yaml

```diff
@@ -0,0 +1,161 @@
+---
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: ab-load
+  labels:
+    app.kubernetes.io/version: "0.1"
+    app.kubernetes.io/name: "ab-load-task"
+    tool: ab
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/displayName: apache-benchmark-load
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/platforms: linux/amd64
+    tekton.dev/tags: ab, apache, benchmark, http, load, test
+spec:
+  description: >-
+    This `task` can be used to start an http load test as part of an application test pipeline.
+
+    Apache benchmark is a simple and powerful tool used to generate http load to emulate a
+    load peak for your http/https exposed application.
+  params:
+    - name: url
+      type: string
+      description: The URL targeted by the load test
+      default: "http://myapp.target_cluster.local/"
```

Comment:
I guess we can remove the default url as it is a required field.

Reply:
changed  in https://github.com/tektoncd/catalog/pull/1002/commits/9c29bc82f0ef066629b0373b3538cf0a20838dde